openmc/tallies.py

```diff
@@ -1410,6 +1432,765 @@ def export_results(self, filename='tally-results', directory='.',
+            new_tally._std_dev = np.abs(new_tally.mean) * \
+                                 np.sqrt(first_term**2 + second_term**2)
+
+        elif isinstance(power, (Integral, Rational)):
+
+            new_tally.name = self.name
+            new_tally._mean = self._mean ** power
+            self_rel_err = self.std_dev / self.mean
+            new_tally._std_dev = np.abs(new_tally._mean * power * self_rel_err)
+            new_tally.estimator = self.estimator
+            new_tally.with_summary = self.with_summary
+            new_tally.num_realization = self.num_realizations
+            new_tally.num_score_bins = self.num_score_bins
+
+            for filter in self.filters:
+                new_tally.add_filter(filter)
+            for nuclide in self.nuclides:
+                new_tally.add_nuclide(nuclide)
+            for score in self.scores:
+                new_tally.add_score(score)
+
+        else:
+            msg = 'Unable to raise Tally ID={0} to ' \
+                  'power {1}'.format(self.id, power)
+            raise ValueError(msg)
+
+        return new_tally
+
+    def __radd__(self, other):
+        """Right addition with a scalar value.
+
+        This reverses the operands and calls the __add__ method.
+
+        Parameters
+        ----------
+        other : Integer or Rational
+            The scalar value to add to this tally
+
+        Returns
+        -------
+        Tally
+            A new derived tally of this tally added with the scalar value.
+
+        """
+
+        return self + other
+
+    def __rsub__(self, other):
+        """Right subtraction from a scalar value.
+
+        This reverses the operands and calls the __sub__ method.
+
+        Parameters
+        ----------
+        other : Integer or Rational
+            The scalar value to subtract this tally from
+
+        Returns
+        -------
+        Tally
+            A new derived tally of this tally subtracted from the scalar value.
+
+        """
+
+        return -1. * self + other
+
+    def __rmul__(self, other):
+        """Right multiplication with a scalar value.
+
+        This reverses the operands and calls the __mul__ method.
+
+        Parameters
+        ----------
+        other : Integer or Rational
+            The scalar value to multiply with this tally
+
+        Returns
+        -------
+        Tally
+            A new derived tally of this tally multiplied by the scalar value.
+
+        """
+
+        return self * other
+
+    def __rdiv__(self, other):
+        """Right division with a scalar value.
+
+        This reverses the operands and calls the __div__ method.
+
+        Parameters
+        ----------
+        other : Integer or Rational
+            The scalar value to divide by this tally
+
+        Returns
+        -------
+        Tally
+            A new derived tally of the scalar value divided by this tally.
+
+        """
+
+        return self * (1. / other)
```

Comment:
We really want other/self, not self/other, no? I suppose you could do `other * self**-1`.